<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="gopal-PC" ignored="3" name="Default test" tests="1" failures="1" timestamp="09 Jul 2019 19:36:35 GMT" time="64.169" errors="0">
  <testcase name="CreateActlinkTest" time="0.144" classname="com.qa.pages.TestCases.SignInPageTests">
    <failure type="java.lang.IllegalArgumentException" message="Sheet index (1) is out of range (no sheets)">
      <![CDATA[java.lang.IllegalArgumentException: Sheet index (1) is out of range (no sheets)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.validateSheetIndex(XSSFWorkbook.java:1532)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.getSheetAt(XSSFWorkbook.java:1139)
	at com.qa.Utility.TestUtil.getCellData(TestUtil.java:42)
	at com.qa.pages.SignInPage.createAct(SignInPage.java:60)
	at com.qa.pages.TestCases.SignInPageTests.CreateActlinkTest(SignInPageTests.java:60)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </failure>
  </testcase> <!-- CreateActlinkTest -->
  <testcase name="verifyCallusNowTextTest" time="0.0" classname="com.qa.pages.TestCases.SignInPageTests">
    <ignored/>
  </testcase> <!-- verifyCallusNowTextTest -->
  <testcase name="VerifySignInLinkTest" time="0.0" classname="com.qa.pages.TestCases.SignInPageTests">
    <ignored/>
  </testcase> <!-- VerifySignInLinkTest -->
  <testcase name="VerifyHomePageLinkTest" time="0.0" classname="com.qa.pages.TestCases.SignInPageTests">
    <ignored/>
  </testcase> <!-- VerifyHomePageLinkTest -->
</testsuite> <!-- Default test -->
